Cargo.toml

```diff
@@ -6,7 +6,7 @@ edition = "2021"
 [dependencies]
 geng = { git = "https://github.com/kuviman/geng" }
 serde = "1"
-noise = "0.7"
+noise = "0.8.2"
 
 [build-dependencies]
-cmake = "<=0.1.45" # https://github.com/PistonDevelopers/freetype-sys/issues/99
\ No newline at end of file
+cmake = "0.1.49" # https://github.com/PistonDevelopers/freetype-sys/issues/99
```

Comment:
Using latest cmake crate dependency is going to break builds with latest locally installed cmake, see issue in the comment, this is a hack until that issue is resolved